extras/claude.sh

```
@@ -0,0 +1,195 @@
+3. Produce plan.best.md — a synthesized plan that takes the strongest elements from each. It must include:
+   - Final root cause analysis
+   - Ordered list of files and functions to change
+   - Concrete implementation steps
+   - Test cases to add or modify
+   - Known risks and mitigations
+4. Do NOT write any code yet."
+
+if [ ! -f plan.best.md ]; then
+  log "Warning: plan.best.md not produced. Falling back to first available plan."
+  first_plan=$(find . -maxdepth 1 -name 'plan.[0-9]*.md' | sort | head -1)
+  [ -n "$first_plan" ] && cp "$first_plan" plan.best.md || {
```

Shellcheck (0.11.0)

[info] Note that A && B || C is not if-then-else. C may run when A is true.

(SC2015)